recipes/limix_util/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "limix_util" %}
+{% set version = "1.0.6" %}
+{% set sha256 = "36cb636d2ab45284868c2c0302f08b99670e64ffcb2be6fb6baa78f06aee10ae" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+  run:
+    - python
+    - pytest
+    - numpy
+    - humanfriendly
+    - asciitree
+    - progressbar2
+    - h5py
+
+test:
+  requires:
+    - pytest
+    - numpy
+    - humanfriendly
+    - asciitree
+    - progressbar2
+    - h5py
+  imports:
+    - limix_util
+  commands:
+    - python -c "import limix_util; limix_util.test()"
+
+about:
+  home: https://github.com/Horta/limix-util
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Utils for LIMIX'
+
+  description: |
+    Utils for LIMIX.
```

Comment:
This isn't required. So if we don't have something better, we could just drop this.